.evergreen/config_generator/etc/sanitizers/test.py

```diff
@@ -55,6 +55,10 @@ def generate_test_tasks(SSL, TAG, MATRIX, MORE_COMPILE_TAGS=None, MORE_TEST_TAGS
 
         for auth, topology, server_ver in product(auths, topologies, server_vers):
             test_tags = tags + [auth, topology, server_ver] + MORE_TEST_TAGS
+            # Do not add `nossl` tag to prevent being selected by legacy config variants.
+            # Remove the `if` when CDRIVER-4571 is resolved.
+            if SSL != 'nossl':
+                test_tags += SSL
```

Comment:
I believe this should be `test_tags += [SSL]` or `test_tags.append(SSL)` to avoid the following result in the generated config:

```
[sanitizers-matrix-asan, ..., server, "4.2", o, p, e, n, s, s, l]
```

Also applies to other instances in this PR.

Reply:
Oof. Thank you for the catch.